bridge_adaptivity/module/forms.py

```diff
@@ -14,6 +14,9 @@ class ActivityForm(ModelForm):
     class Meta:
         model = Activity
         exclude = ['collection', 'lti_consumer']
+        widgets = {
+            'type': forms.HiddenInput(),
```

Comment:
We've decided to name this parameter as `stype`

Reply:
in model it's already has name stype. forgot to change it here...